valhalla_build_admins error during planet import

Just tried to build admins from master commit f2b85b1a811c1ca2bc50a951340d3710be5be530. It's finished with sqlite3 error. See log below.

```
evgen@vmap1:~/data$ valhalla_build_admins --config valhalla.json planet-latest.osm.pbf
2017/08/07 08:35:41.268476 [INFO] Parsing files: planet-latest.osm.pbf
2017/08/07 08:35:41.268580 [INFO] Parsing relations...
2017/08/07 08:50:59.130250 [INFO] Finished with 3223 admin polygons comprised of 222807 ways
2017/08/07 08:50:59.130524 [INFO] Parsing ways...
2017/08/07 09:08:54.892965 [INFO] Finished with 108453 ways comprised of 9624012 nodes
2017/08/07 09:08:54.892990 [INFO] Parsing nodes...
2017/08/07 09:09:09.878602 [INFO] Processed 500000 nodes on ways
2017/08/07 09:09:42.508932 [INFO] Processed 1000000 nodes on ways
2017/08/07 09:10:01.732508 [INFO] Processed 1500000 nodes on ways
2017/08/07 09:10:20.005931 [INFO] Processed 2000000 nodes on ways
2017/08/07 09:10:35.650092 [INFO] Processed 2500000 nodes on ways
2017/08/07 09:11:11.318891 [INFO] Processed 3000000 nodes on ways
2017/08/07 09:11:50.056814 [INFO] Processed 3500000 nodes on ways
2017/08/07 09:12:07.724923 [INFO] Processed 4000000 nodes on ways
2017/08/07 09:12:36.931221 [INFO] Processed 4500000 nodes on ways
2017/08/07 09:13:06.955092 [INFO] Processed 5000000 nodes on ways
2017/08/07 09:13:30.245691 [INFO] Processed 5500000 nodes on ways
2017/08/07 09:14:16.567837 [INFO] Processed 6000000 nodes on ways
2017/08/07 09:14:57.348323 [INFO] Processed 6500000 nodes on ways
2017/08/07 09:15:30.612172 [INFO] Processed 7000000 nodes on ways
2017/08/07 09:15:57.651610 [INFO] Processed 7500000 nodes on ways
2017/08/07 09:16:28.540962 [INFO] Processed 8000000 nodes on ways
2017/08/07 09:16:52.804094 [INFO] Processed 8500000 nodes on ways
2017/08/07 09:17:30.088763 [INFO] Processed 9000000 nodes on ways
2017/08/07 09:18:19.473794 [INFO] Processed 9500000 nodes on ways
2017/08/07 09:26:03.354485 [INFO] Finished with 9512419 nodes
2017/08/07 09:26:06.607812 [INFO] Created admin access table.
2017/08/07 09:26:06.612250 [INFO] Created admin table.
2017/08/07 09:31:56.527629 [INFO] Inserted 3147 admin areas
2017/08/07 09:31:56.802306 [INFO] Created spatial index
2017/08/07 09:31:56.805022 [INFO] Created Level index
2017/08/07 09:31:56.807477 [INFO] Created Drive On Right index
2017/08/07 09:33:37.203593 [INFO] Done updating drive on right column.
2017/08/07 10:02:07.474312 [INFO] Done updating Parent admin
2017/08/07 10:02:07.480867 [ERROR] sqlite3_step() error: NOT NULL constraint failed: admin_access.admin_id.  Ignore if not using a planet extract.
2017/08/07 10:02:07.675427 [INFO] Finished.
```

@gknisely Thanks! I just checked France boundary and it's seems fixed already, but that's fix is not included into latest planet file yet. Right now i'm in the middle of routing tile generation. I'll check again with updated planet later.

May be it would be useful to print relation number that caused the error.
`2017/08/07 10:02:07.480867 [ERROR] sqlite3_step() error: NOT NULL constraint failed: admin_access.admin_id.  Ignore if not using a planet extract.
` is not very informative.

--- a/src/mjolnir/valhalla_build_admins.cc
+++ b/src/mjolnir/valhalla_build_admins.cc
@@ -459,6 +459,8 @@ void BuildAdminFromPBF(const boost::property_tree::ptree& pt,
         std::unique_ptr<Geometry> mline (gf->createMultiLineString(lines.release()));
         std::vector<std::string> wkts = GetWkts(mline);
         std::string name;
+        std::string name_en;
+        std::string iso;
 
         for (const auto& wkt : wkts) {
 
@@ -468,8 +470,8 @@ void BuildAdminFromPBF(const boost::property_tree::ptree& pt,
           sqlite3_bind_int (stmt, 1, admin.admin_level());
 
           if (admin.iso_code_index()) {
-            name = osmdata.name_offset_map.name(admin.iso_code_index());
-            sqlite3_bind_text (stmt, 2, name.c_str(), name.length(), SQLITE_STATIC);
+            iso = osmdata.name_offset_map.name(admin.iso_code_index());
+            sqlite3_bind_text (stmt, 2, iso.c_str(), iso.length(), SQLITE_STATIC);
           }
           else
             sqlite3_bind_null(stmt,2);
@@ -480,8 +482,8 @@ void BuildAdminFromPBF(const boost::property_tree::ptree& pt,
           sqlite3_bind_text (stmt, 4, name.c_str(), name.length(), SQLITE_STATIC);
 
           if (admin.name_en_index()) {
-            name = osmdata.name_offset_map.name(admin.name_en_index());
-            sqlite3_bind_text (stmt, 5, name.c_str(), name.length(), SQLITE_STATIC);
+            name_en = osmdata.name_offset_map.name(admin.name_en_index());
+            sqlite3_bind_text (stmt, 5, name_en.c_str(), name_en.length(), SQLITE_STATIC);
           }
           else
             sqlite3_bind_null(stmt,5);

